xbmc/addons/AddonStatusHandler.cpp

```diff
@@ -102,10 +102,7 @@ void CAddonStatusHandler::Process()
       pDialog->SetHeading(CVariant{heading});
       pDialog->SetLine(1, CVariant{24070});
       pDialog->SetLine(2, CVariant{24073});
-
-      //send message and wait for user input
-      ThreadMessage tMsg = {TMSG_DIALOG_DOMODAL, WINDOW_DIALOG_YES_NO, g_windowManager.GetActiveWindow()};
-      CApplicationMessenger::Get().SendMessage(tMsg, true);
+      pDialog->Open();
```

Comment:
we check for the main thread in Open() and if not we send a thread msg which should be the same like this, or isn't it?